### ISSUE TITLE
Support for Firefox

### ISSUE THREAD

**Issue by nikogt4** [radiance-project/ear-web#23]
Would love to see the app on Firefox :) thank you devs 🙏

**Comment by Kipp-ie** [comment]
I don't think that will work, chromium based browsers (Chrome, Edge, Brave etc) support the Webserial API (What Firefox doesn't). When Firefox adds webserial, this project will probably work on firefox.

**Comment by Lisra-git** [comment]
Hey,
As said above this is more of a Firefox issue which doesn't support Webserial for now.
Hopefully, things could change at some point, fingers crossed!

**Comment by krystian3w** [comment]
Apparently, minimal support will be available in a month: https://bugzilla.mozilla.org/show_bug.cgi?id=2029625 (In ESR in about two months), so it will be interesting to see what issues can already be detected in the beta.

**Comment by kinoosaki** [comment]
@krystian3w it already tries to install an add-on or something when I connect only to fail because of ear (web), that looks like webserial might be working now

<img width="664" height="644" alt="Image" src="https://github.com/user-attachments/assets/fa0f81ff-9711-4d74-86e1-35400bb6cb38" />

**Comment by krystian3w** [comment]
From a layperson's perspective, the only thing left to do is to emulate the Bluetooth API so that the official WebSerial API can be used with it.
